executor/runtime/docker/docker_env.go

```diff
@@ -23,6 +24,13 @@ export {{ $key }}={{ $val | escape_sq }}
 export {{ $key }}={{ escapeSQWithFallback $key $val }}
 {{ end -}}
 {{ end -}}
+
+{{ if .InvalidEnv }}
+# Any extra invalid env variables will be commented out here:
+{{ range $key, $val := .InvalidEnv -}}
+# export {{ $key }}={{ $val | escape_sq }}
```

Comment:
Left over?

Reply:
What do you mean?
It is left over in the sense that I didn't want to silently discard them?

Reply:
I meant this line `# export {{ $key }}={{ $val | escape_sq }}`.

Reply:
Yea, line 31? That is commented out in the template, so that when someone goes to look at the profile.sh file they will see "why" those env variables are not set (they are commented out)

Reply:
Ah, I was misreading. Cool.